migrations/106_exams_add_PS_columns.sql

```diff
@@ -0,0 +1,11 @@
+/* From PS exams as of 2018-02-24 */
+
+ALTER TABLE exams ADD COLUMN IF NOT EXISTS exam_string text NOT NULL;
```

Comment:
For every line like this we also need to update the corresponding entry in `PrairieLearn/database/tables/exams.pg`. The files in `database/` describe what the end state of the DB schema should look like. The test suite checks that if we take a blank DB and run all the migrations on it, then what we get is indeed the same as `database/`. This is the first check when you do `npm test`.